libclc/amdgcn-amdhsa/libspirv/math/ldexp.cl

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <clcmacro.h>
+#include <config.h>
+#include <math/math.h>
+#include <spirv/spirv.h>
+
+double __ocml_ldexp_f64(double, int);
+float __ocml_ldexp_f32(float, int);
+
+#define __CLC_FUNCTION __spirv_ocl_ldexp
+#define __CLC_BUILTIN __ocml_ldexp
+#define __CLC_BUILTIN_F __CLC_XCONCAT(__CLC_BUILTIN, _f32)
+#define __CLC_BUILTIN_D __CLC_XCONCAT(__CLC_BUILTIN, _f64)
+
```

Comment:
Are these defines needed?
I suppose `_CLC_DEFINE_BINARY_BUILTIN` does the job.

Please, check if the same defines can be removed from `libclc/amdgcn-amdhsa/libspirv/math/frexp.cl`, `libclc/amdgcn-amdhsa/libspirv/math/modf.cl` and `libclc/amdgcn-amdhsa/libspirv/math/sincos.cl` as well.

Reply:
Thank you, I will check now.